tools/make_dictionary_file.py

```diff
@@ -0,0 +1,65 @@
+"""
+Makes dictionary file from mozc
+(https://github.com/google/mozc/tree/master/src/data/dictionary_oss)
+"""
+
+def make_jp_name_dictionary(input_file_names):
+    """
+    Makes japanese name dictionary.
+
+    Args:
+        input_file_names: a list of file names
+
+    Out put format:
+        kanji '\t' yomigana(hiragana) '\n'
+        kanji '\t' yomigana(hiragana) '\n' ...
+    """
+
+    yomigana_list = []
+    for input_file_name in input_file_names:
+
+        with open(input_file_name, 'r') as f:
+            for line in f:
+                line = line.rstrip()
+                splited_line = line.split('\t')
+                left = int(splited_line[1])
+                right = int(splited_line[2])
+
+                if ((left == 1845 and right == 1845) or
```

Comment:
what are these numbers?
consider putting them in a list, that you can put as an argument to the function, so you can reuse this function for different numbers